WordPress/Classes/Utility/App Configuration/AppColor.swift

```diff
@@ -91,30 +87,41 @@ struct UIAppColor {
 
 #if IS_JETPACK
     static let tint = UIColor.label
+
     static let brand = UIColor(light: CSColor.JetpackGreen.shade(.shade40), dark: CSColor.JetpackGreen.shade(.shade30))
 
     static func brand(_ shade: ColorStudioShade) -> UIColor {
         CSColor.JetpackGreen.shade(shade)
     }
+
+    static let primary = CSColor.JetpackGreen.base
+
+    static func primary(_ shade: ColorStudioShade) -> UIColor {
+        CSColor.JetpackGreen.shade(shade)
+    }
 #endif
 
 #if IS_WORDPRESS
     static let tint = brand
+
     static let brand = CSColor.WordPressBlue.base
 
     static func brand(_ shade: ColorStudioShade) -> UIColor {
         CSColor.WordPressBlue.shade(shade)
```

Comment:
There is a bit of a mess with this naming, and it might be worth refining it later.

- `brand` is a confusing name because A8C has multiple brands and wpios sometimes uses Jetpack brand colors
- `WordPressBlue` is a confusing name because it's actually "WordPress.Com Blue"
- `primary` is a confusing name because in SwiftUI `Color.primary` typically means primary text color (black/white)
- `tint` is a confusing name because jpios sometimes uses `.label` and sometimes uses Jetpack green as a tint color

I don't have suggestions how to solve it because a lot of this complexity isn't incidental.